bundle/jsky.app.ot/src/main/scala/jsky/app/ot/gemini/parallacticangle/ParallacticAngleControls.scala

```diff
@@ -194,7 +197,6 @@ class ParallacticAngleControls extends GridBagPanel with Publisher {
     for {
       e              <- editor
       ispObservation <- Option(e.getContextObservation)
-      if ObsTargetCalculatorService.targetCalculation(ispObservation).isDefined
```

Comment:
From what I can tell, this filter would keep anything from happening if we could not determine a `Site`, scheduling block, or target component.  Scheduling block is included below but now it looks like we'll continue anyway even if there is no known `Site` or target component?  It looks like that might be okay since the `parallacticAngle` will be `None` in that case and it will display "Target not visible" which seems reasonable.  Is that the reasoning behind this?